src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -344,27 +344,20 @@ public object GetParameterValueByName(string parameterName)
         }
 
         /// <summary>
-        /// Override the element's color in the active view.
+        ///     Override Elements Graphics Settings in Active View.
         /// </summary>
-        /// <param name="color">The color to apply to a solid fill on the element.</param>
-        public Element OverrideColorInView(Color color)
```

Comment:
mhmmmm, I thought that the next release was going to be Dynamo 2.0, and API breaking changes were allowed....actually, just kidding. I had no clue that you were worried about this. I can bring that old clunker back so long as there is room for this Corvette. :-P